package.json

```diff
@@ -44,7 +44,7 @@
         "typesense": "^0.6.1"
     },
     "scripts": {
-        "test": "eslint --ext .json --ext .js --ext .ts && ts-node-transpile-only src/test-records.ts",
+        "test": "eslint --ext .json --ext .js --ext .ts && ts-node --files src/test-records.ts",
```

Comment:
Do you actually want to make this change?

Reply:
No, I'll remove it. :D